x/blob/types/payforblob.go

```diff
@@ -49,7 +50,7 @@ func NewMsgPayForBlobs(signer string, blobs ...*blob.Blob) (*MsgPayForBlobs, err
 	if err != nil {
 		return nil, err
 	}
-	commitments, err := inclusion.CreateCommitments(blobs)
+	commitments, err := inclusion.CreateCommitments(blobs, merkle.HashFromByteSlices, appconsts.DefaultSubtreeRootThreshold)
```

Comment:
[Question] I see sometimes we retrieve the `SubtreeRootThreshold` based on the app version `subtreeRootThreshold := appconsts.SubtreeRootThreshold(version)`  and sometimes we opt in the default value, made me wonder what is the rationale? It seems to me that they should be retrieved similarly all based on the app version.

Reply:
How I've done it is that most tests should just use the latest version by default. Anything touching the state machine should be versioned. 

This part should probably be versioned. I will change it. I was just trying to reduce how much of the API I broke. It seems that a lot of these types weren't as prepared for multi versioning